data/Archlinux.yaml

```diff
@@ -3,3 +3,7 @@ prometheus::version: '2.2.0'
 prometheus::install_method: 'package'
 prometheus::bin_dir: '/usr/bin'
 prometheus::env_file_path: '/etc/default'
+prometheus::node_exporter::package_name: 'prometheus-node-exporter'
```

Comment:
@sebastianrakel Why did you change the value of the parameter ```package_name```? This is part of the download URL and I do not think that ArchLinux uses different binaries. If you remove line 6 and 8, the tests should pass.

Reply:
Because archlinux provides native packages and that one is named `prometheus-node-exporter`.

Reply:
@bastelfreak Ah. I did not know that. In that case we need an additional case for the tests like @sebastianrakel did for ```service_name```.